templates/fine-tune-llm_v2/training_configs/custom/meta-llama/Meta-Llama-3-8B/lora/4xA10-512.yaml

```diff
@@ -41,6 +41,16 @@ deepspeed:
 worker_resources:
   anyscale/accelerator_shape:4xA10G: 0.001
 
+# Liger kernel configuration
+liger_kernel:
+  enabled: True
+  kwargs:
+    rms_norm: True
+    rope: True
+    swiglu: True
+    cross_entropy: True
+    fused_linear_cross_entropy: False
+
```

Comment:
@erictang000 umm did this value change? why was this false again?

Reply:
discussed on slack, for lower context length + batch size, regular cross entropy can be faster and memory is similar. Example w/ this toggled for llama-3.2-1b. But easier to just use defaults w/ liger, so we can turn fused linear cross entropy on in our default configs.
![image](https://github.com/user-attachments/assets/977dd381-ce97-477d-9c69-a9d2760420a8)
![image](https://github.com/user-attachments/assets/10655dce-33e6-4e18-b87a-1ba081223d4e)